src/topic_training/src/Demo.cpp

```diff
@@ -0,0 +1,136 @@
+/* Include all headers of the ROS system we are using 
+   separately. */
+#include "ros/init.h"
+#include "ros/node_handle.h"
+#include "ros/rate.h"
+#include "ros/subscriber.h"
+#include "ros/publisher.h"
+
+/* Include the message type we will be publishing. */
+#include "std_msgs/String.h"
+
+/* Include Stringstream to be able to manipulate the 
+   data we are sending and receiving. (Optional: only 
+   if the message is of type String)*/
+#include <sstream>
+
+/* C++ does not like "magic numbers," so it is better 
+   if we make them variables. */
+const int QUEUE_SIZE = 1000;
```

Comment:
Do we need 1000 messages?  Probably fine to just use 1 or 10.

Reply:
Fixed in 47e0938.